web/src/portico/team.js

```diff
@@ -36,10 +36,6 @@ function get_profile_url(contributor, tab_name) {
 
     const email = contributor.email;
 
-    if (tab_name) {
-        return `https://github.com/zulip/${tab_name}/commits?author=${email}`;
-    }
```

Comment:
I think the bug here is actually that `get_profile_url` should be passing this parameter in the click handler where we are looking at a specific tab, i.e. in the block containing `repo_name_to_tab_name`.

(Fixing this would mean that say, in the "Mobile" tab, the link for Tomasz Kolek on zulip.com/team would correctly go to his contributions to that repository, not zulip/zulip).

Generally anytime you see unused conditionals this, one shouldn't just delete it -- one should consider the other possibly bug (that someone wrote it for a purpose, but forgot to actually pass the parameter)

Reply:
Yeah, you are write there is a missing argument in one of the function calls, will fix that soon and push it. Thanks for spotting that. :heart: